roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -18,20 +18,15 @@
        version="{{ ngi_rnaseq_version }}"
        force=yes
 
-- name: Install NGI-RNAseq
```

Comment:
It was never required. NGI-RNAseq is a nextflow application.
I figured the first revision of the recipe wouldn't work anyway, so I wasn't too keen on doublechecking everything.